CMakeLists.txt

```diff
@@ -73,6 +73,14 @@ set(GOOGLEAPIS_CPP_PROTO_FILES
     "google/bigtable/admin/v2/table.proto"
     "google/bigtable/v2/bigtable.proto"
     "google/bigtable/v2/data.proto"
+    "google/cloud/bigquery/v2/model_reference.proto"
+    "google/cloud/bigquery/v2/standard_sql.proto"
+    "google/cloud/bigquery/v2/model.proto"
```

Comment:
Now I see:
https://github.com/googleapis/googleapis/blob/master/google/cloud/bigquery/v2/encryption_config.proto
also datatransfer and logging directory have some protos.

in googleapis repo. Does it make sense to include them too?

Reply:
Argh.  I was looking at an older version of googleapis/googleapis.  Yes, we probably need to update and add any new protos.